test-cases/tests/upgrade/n10-verify-quota-feature-upgrade.md

```diff
@@ -80,23 +80,21 @@ echo "https://$(oc get route grafana-route -n redhat-rhoam-observability -o=json
 
 9. Compare the values of resources with what you get by running this command from the terminal
 
-> Note: you might encounter `no matches found` errors if using zsh, it is recommended to use bash.
-
 ```bash
 backend_listener_replicas=$(oc get dc backend-listener -n redhat-rhoam-3scale --no-headers=true | awk '{print $4}')
-backend_listener_resources=$(oc get dc backend-listener -n redhat-rhoam-3scale -o json | jq -r .spec.template.spec.containers[0].resources)
+backend_listener_resources=$(oc get dc backend-listener -n redhat-rhoam-3scale -o json | jq -r '.spec.template.spec.containers[0].resources')
```

Comment:
Adding this as a fyi. To get this information the dependency on `jq` can be removed by using jsonpath in the oc command. The above example would be.
```
 oc get dc apicast-production -n redhat-rhoami-3scale -o jsonpath='{.spec.template.spec.containers[0].resources}{"\n"}'
```
One possible unwanted side affect is the removal of the pretty printing that yq does. Of course that can be regained by piping into yq after the result has being returned. 
```
oc get dc apicast-production -n redhat-rhoami-3scale -o jsonpath='{.spec.template.spec.containers[0].resources}{"\n"}' | jq
```
Personal I think it would be better to use `jsonpath` and remove external dependences as much as possible.